rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,27 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+author: Ömer Günal, oscd.community
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: low
+logsource:
+    product: linux
+detection:
```

Comment:
could you please add a reference to the [relevant Atomic Red Team test](https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1553.004/T1553.004.md)?

Reply:
Thanks a lot!